README_android.txt

```diff
@@ -19,26 +18,21 @@ Install the SDK
 ===============
 
 
-Start by extracting the SDK. You then need to add the SDK to your PATH
-environment variable. If you installed the SDK in $HOME/android-sdk, add
-$HOME/android-sdk/tools to your PATH.
+The most simple way is to install Android Studio which by default will
+place a copy of the SDK into ~/Android/Sdk.
 
-Once extracted you must run "android" (or android.bat on Windows) and install
-a version of the SDK. Allegro needs version 13 (3.1) of the SDK or higher to
-build. It can run on version 9 and up still though, if you include the
-correct uses-sdk fields in your manifest. See:
-https://developer.android.com/guide/topics/manifest/uses-sdk-element.html
-
-Also make sure the JAVA_HOME environment variable is set to your JDK
-directory (not always the case.)
+Alternatively you can also download the command-line SDK tools.
```

Comment:
Probably should mention ALLEGRO_HOME enviroment variable here.

Also, I had to accept the licenses before it would let me build, maybe add that as an optional step:

~/Android/Sdk/tools/bin/sdkmanager --licenses